penguin/web/middleware/auth.py

```
@@ -105,137 +181,68 @@ async def dispatch(self, request: Request, call_next):
+        token = extract_bearer_token(request)
         if token:
-            claims = self._validate_jwt(token)
+            claims = validate_jwt(token, self.config)
             return {
                 "method": "jwt",
                 "subject": claims.get("sub", "unknown"),
                 "metadata": claims
             }
 
-        # No valid authentication found
         raise AuthenticationError("No valid authentication credentials provided")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)